### PR TITLE
[#92] Created endpoint to get captions associated to a moment

### PR DESCRIPTION
**Pull request by tehblasian**
Returns captions associated to a moment, sorted by most recent.

### CODE REVIEW

File: backend/app/routes/captions.js [hunk 1]
```diff
@@ -30,6 +30,68 @@ const createCaption = {
     },
 };
 
+const getCaptionsByMoment = {
+    method: 'GET',
+    path: '/api/captions',
+    handler: (request, reply) => {
+        // If not authorized
+        if (!request.auth.credentials) {
+            return reply.response({ code: 4 }).code(401);
```
Comment (ZacharyBys) [inline backend/app/routes/captions.js]: We don't need to be logged in to view a moments captions

**Review by justin-cotarla — CHANGES_REQUESTED** [review]
No garbage data on bad codes

**Review by justin-cotarla — CHANGES_REQUESTED** [review]
Make your .then()'s beautiful

**Review by justin-cotarla — CHANGES_REQUESTED** [review]
Put the ID and the username in a user attribute for the response

**Review by justin-cotarla — CHANGES_REQUESTED** [review]
Remove debug logs; use ES6 const

**Review by justin-cotarla — APPROVED** [review]
Looks good. Going to make getting captions great again.

**Pull request merged**